### PR TITLE
docs: Add hint how to install vscode

### PR DESCRIPTION
**Pull request by simone-weiss**
Only a small change, but one can not assume that people
    use vscode already.

# Definition of Done

The PR shall be merged only if all items mentioned in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been followed. In case an item is not applicable as described, please provide a short explanation in the description.

- [x] All steps in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been handled

### CODE REVIEW

**Comment by simone-weiss** [comment]
- Note also the svgs are not displayed good if one uses a dark github setting and checks from the markdown files.

**Review by windsource — APPROVED** [review]
LGTM 👍

File: doc/docs/development/build.md [hunk 1]
```diff
@@ -6,7 +6,10 @@ The repo provides a Visual Studio Code [dev container](https://code.visualstudio
 
 ### Prerequisites
 
+As prerequisites, you need to have the following tools set up:
+
 - Docker ([Installation instructions](https://docs.docker.com/engine/install))
+- Vscode ([Installation instructions](https://code.visualstudio.com/download))
```
Comment (windsource) [inline doc/docs/development/build.md:12]:
```suggestion
- Visual Studio Code ([Installation instructions](https://code.visualstudio.com/download))
```

**Review by windsource — CHANGES_REQUESTED** [review]
Only small change required

File: doc/docs/development/build.md (hunk 1 above)
Comment (simone-weiss) [reply doc/docs/development/build.md]: Good point - done

**Comment by windsource** [comment]
> * Note also the svgs are not displayed good if one uses a dark github setting and checks from the markdown files.

Thanks for the hint. I have created a #187 for that.

**Review by windsource — APPROVED** [review]
OK

**Pull request merged**